sys-block/parted/parted-3.2.recipe

```diff
@@ -77,36 +80,42 @@ BUILD_PREREQUIRES="
 	cmd:autoreconf
 	cmd:awk
 	cmd:gcc$secondaryArchSuffix
-	cmd:gettext
+	cmd:gettext$secondaryArchSuffix
 	cmd:gperf
 	cmd:grep
 	cmd:iconv
 	cmd:ld$secondaryArchSuffix
 	cmd:libtoolize$secondaryArchSuffix
 	cmd:make
-	cmd:makeinfo
 	cmd:pkg_config$secondaryArchSuffix
 	cmd:sed
 	"
 
+TEST_REQUIRES="
+	cmd:dd
+	"
+
 defineDebugInfoPackage parted$secondaryArchSuffix \
 	"$libDir"/libparted.so.$libVersion \
 	"$libDir"/libparted-fs-resize.so.$libFsResizeVersion
 
 BUILD()
 {
 	autoreconf
+	MAKEINFO=true \
 	runConfigure --omit-dirs sbinDir ./configure --sbindir=$commandBinDir \
 		--disable-device-mapper
-	make $jobArgs
+	make $jobArgs \
+		CPPFLAGS="-Wimplicit-fallthrough=0" \
+		LDFLAGS="-lblkid -luuid -lintl" \
```

Comment:
It's not a good practice to override flags with `make`, as it will override any custom flags that upstream use. I would recommend changing the variables before running `configure`, which would pick them up and appends them accordingly.

Reply:
Thanks @alaviss. I'll fix this.
I'll also compare the resulting runtime, and in case it does not change then we can keep `REVISION` unchanged because the recipe is marked as untested on all architectures.

Reply:
→ #2616